mcsema/cfgToLLVM/JumpTables.cpp

```diff
@@ -229,7 +229,17 @@ static void doJumpTableViaSwitch(TranslationContext &ctx, llvm::BasicBlock *&blo
   int myindex = 0;
   for (auto jmp_block_va : jmpblocks) {
     BasicBlock *toBlock = ctx.va_to_bb[jmp_block_va];
-    TASSERT(toBlock != NULL, "Could not find block");
+     
+    // FIXME: this needs more testing, I (Josh) added it and I'm not 100% on the syntax, but it's worked thus far
```

Comment:
Use format `TODO(josh2059): ..` or `TODO(josh): ...`. This way the creator of the TODO is unambiguous, and people will know who to ask for clatification. Avoid qualified personal pronouns.